packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -87,8 +86,9 @@ contract TaikoL1 is EssentialContract, ITaikoL1, TaikoEvents, TaikoErrors {
 
         (meta_, deposits_) = LibProposing.proposeBlock(state, tko, config, this, _params, _txList);
 
-        if (!state.slotB.provingPaused) {
-            LibVerifying.verifyBlocks(state, tko, config, this, config.maxBlocksToVerifyPerProposal);
+        uint256 freq = config.verificationFrequencyFactor;
```

Comment:
I think the `freq` (or rather `config.verificationFrequencyFactor`) variable shall has the same value as `maxBlocksToVerify`, so we can reuse.
Why ?

Let's say `verificationFrequencyFactor` is 16 but `maxBlocksToVerify` 8. In a world where proposing and verification are roughly equal as PR desc. suggest, it would mean that the verification will be a bottleneck, as we only trying to verify 8, after every 16 block proposed (or proven).